README.md

```diff
@@ -19,29 +19,15 @@ This site uses a Jekyll, a Ruby framework. You'll need Ruby and Bundler installe
 
 cd into the directory where you cloned this repository, then install the required gems with `bundle install`. This will automatically put the gems into `./vendor/bundle`.
 
-Start the server in the context of the bundle:
+Start the server in the context of the bundle (use `-I` for incremental builds):
 
-    bundle exec jekyll serve
+    bundle exec jekyll serve -I
 
-The generated site is available at `http://localhost:4000`
-
-If you add `--watch` at the end of the command line above, Jekyll will automatically watch for changes on the filesystem, and regenerate the site. It can take a few minutes for your changes to appear. Watch Jekyll's output. When you start up you'll see this:
-
-     $ bundle exec jekyll serve --watch
-     Configuration file: /Users/ben/src/scala.github.com/_config.yml
-                 Source: /Users/ben/src/scala.github.com
-            Destination: /Users/ben/src/scala.github.com/_site
-          Generating...
-                     done.
-    Auto-regeneration: enabled for '/Users/ben/src/scala.github.com'
+`It might take around 5 minutes at first but incremental compilations will be fast.`
```

Comment:
I added it as indication for users, that first build might take a while. I can change it if you think it's not a good idea.